### DISCUSSION TITLE
Suggestion: add two formulas for maldetect and tiger

### DISCUSSION THREAD

**Discussion by Zorono** [Homebrew/discussions#3196] [Writing Formulae/Casks] [answered]
Hello every Creature,
it'd be great if someone added formulas for maldetect and tiger (please note that I don't have any experience with ruby or formula creations...)
maldetect: http://www.rfxn.com/downloads/maldetect-current.tar.gz (or https://github.com/rfxn/linux-malware-detect)
tiger: https://www.nongnu.org/tiger/ (https://git.savannah.gnu.org/cgit/tiger.git)

**Comment by SMillerDev** [comment, marked as answer]
https://docs.brew.sh/Adding-Software-to-Homebrew
